eng/Version.Details.xml

```diff
@@ -3,17 +3,17 @@
   <ProductDependencies>
   </ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="9.0.0-beta.23531.4">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="9.0.0-beta.23551.1">
```

Comment:
I had a question about the version to use. Why did you decide to upgrade arcade 9.0 instead of 8.0? The uber issue suggested updated to 8.0 https://github.com/dotnet/maintenance-packages/issues/16#issue-1936330544

Reply:
Why stay on the 8.0 Arcade? 9.0 is newer and better.

Reply:
Right. But I was wondering if it was because this is a servicing branch so we want to use the latest stable version we use in servicing, which would 8.0. But if that's not a problem with arcade, and we prefer 9.0, I don't mind.

Reply:
IMO this repo's main branch is not intended to be a servicing branch. It should follow the latest available toolset, infrastructure and officially supported TFMs.